src/tablo/Column/Column.js

```diff
@@ -41,7 +41,9 @@ const Column = (args) => {
 
   const cell = ({ rowIndex, columnKey }) => {
     const elem = find([].concat(children), child => child.type === Cell) || defaultCell;
-    return React.cloneElement(elem, { data: (data[rowIndex] || {})[columnKey] });
+    const rowData = data[rowIndex] || {};
+    const dataCell = rowData[columnKey];
+    return React.cloneElement(elem, { data: dataCell, rowData, rowIndex });
```

Comment:
Not a strong opinion about `tableData`/`allData` but I just thought that with `rowIndex` one could easily overcome the needs (not typical) to have all the data as argument.

The `columnIndex`/`columnKey` is defined by the user in the `Column.name`

```js
const col = <Column name='columnName'><Cell>{renderData}</Cell></Column>
function renderData(cellData, ...args) { return cellData; }// cellData is `data[rowIndex]['columnName']
```